libraries/AP_ExternalAHRS/AP_ExternalAHRS_VectorNav.h

```diff
@@ -63,20 +63,23 @@ class AP_ExternalAHRS_VectorNav : public AP_ExternalAHRS_backend {
 
     void initialize();
 
-    void process_ins_packet1(const uint8_t *b);
-    void process_ins_packet2(const uint8_t *b);
-    void process_ahrs_packet(const uint8_t *b);
     void run_command(const char *fmt, ...);
+    void process_imu_packet(const uint8_t *b);
+    void process_ahrs_ekf_packet(const uint8_t *b);
+    void process_ins_ekf_packet(const uint8_t *b);
+    void process_ins_gnss_packet(const uint8_t *b);
 
     uint8_t *pktbuf;
     uint16_t pktoffset;
     uint16_t bufsize;
 
-    struct VN_INS_packet1 *last_ins_pkt1;
-    struct VN_INS_packet2 *last_ins_pkt2;
+    struct VN_imu_packet const *latest_imu_packet = nullptr;
+    struct VN_INS_ekf_packet const *latest_ins_ekf_packet = nullptr;
+    struct VN_INS_gnss_packet const *latest_ins_gnss_packet = nullptr;
```

Comment:
```suggestion
    struct VN_imu_packet const *latest_imu_packet;
    struct VN_INS_ekf_packet const *latest_ins_ekf_packet;
    struct VN_INS_gnss_packet const *latest_ins_gnss_packet;
```

Reply:
Is it improper to initialize to nullptr?

Reply:
You get it for free in AP, doing it manually just costs flash.

Reply:
Oh interesting, didn't know that. Thanks! Will update